src/mainboard/clevo/adl-p/vboot-rwa.fmd

```diff
@@ -0,0 +1,40 @@
+FLASH@0xff000000 0x1000000 {
```

Comment:
ADL models have a 32MB flash. You can change this to:

```suggestion
FLASH 32M {
```

Reply:
Changed.